examples/applier/apply-some-yaml/main.go

```diff
@@ -0,0 +1,113 @@
+//Package main: This program shows how to create resources based on yamls template located in
+//the different directory or bindata path or array of string.
+package main
+
+import (
+	"flag"
+	"os"
+
+	// "github.com/open-cluster-management/library-go/examples/applier/bindata"
+	"github.com/open-cluster-management/library-go/pkg/applier"
+	libgoclient "github.com/open-cluster-management/library-go/pkg/client"
+	"k8s.io/klog"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func usage() {
+	klog.Infof("Usage: apply-some-yaml -k kubeconfig\n")
+	flag.PrintDefaults()
+}
+
+func showUsageAndExit(exitcode int) {
+	usage()
+	os.Exit(exitcode)
+}
```

Comment:
why these 2 different functions. can combine these two

Reply:
I will, it comes for google examples

Reply:
No seems to be needed to assign flag.Usage